### PR TITLE
Connect DB and GraphQL in spoken languages

### LINKED ISSUE #45: Support Language in different locales
We may need to adjust the API as well as the DB model.

Let me write this the way I see it in Postgres and then we'll discuss how to best incorporate it into GraphQL.

 My earlier comment:

>  Can we go back to the original data model I proposed for this? There is an intersection of Locale and Language that makes this more complicated.
> 
> - We need to show which languages a person speaks
> - When the site's locale changes, will will need to display the language name in the proper locale. 
> 
> E.g. the site will say '日本語'　or 'Japanese" for a Japanese-speaking practitioner depending on locale
> The two letter ISO code 'ja' will be the identifier.

_Originally posted by @ann-kilzer in https://github.com/ourjapanlife/findadoc-server/pull/26#discussion_r1051554172_

### PR DESCRIPTION
**Pull request by ann-kilzer**
Part of #87 and resolves #45

# What changed
- [x] Add SpokenLanguages graphQL query
- [x] connect to Prisma
- [x] fix DB model to include native language
- [x] Remove "language" enum from GraphQL
- [x] Query a single by iso code

# Testing instructions

## Select Many
Run the following in Apollo Sandbox:
```
query Query {
  spokenLanguages {
    iso639_3
    nameEn
    nameJa
    nativeName
  }
}
```

## Select one
Run the backend then click the following to fill the query

https://studio.apollographql.com/sandbox/explorer?endpoint=http%3A%2F%2Flocalhost%3A3001%2F&explorerURLState=N4IgzghgbgpgJgYQPYBsUwMYBcCWSB2AknCAFwgwQBGcGA7DDALQBsEAHAGZMAsdEzKuwCszAAwAmKgGYe0sTwh0JEkABpw0eMjSZcBAKL4sAJwCexMiACM9aROlUWTKhjjte1uh44SAnEzs7BCy0hgS7HCc0iAAvkA&referrer=operation_collections

Test cases for iso code: 'spa' should retrieve spanish, 'eng' english, 'xxx' null

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ module.exports = {
         'arrow-spacing': 'error',
         'block-spacing': ['error', 'always'],
         'brace-style': ['error', '1tbs', { allowSingleLine: true }],
-        camelcase: 'error',
+        camelcase: ['error', {allow: ['639_3']}],
```
Comment (ann-kilzer) [inline .eslintrc.js:68]: allow this pattern as an exception to the camelcase rule

File: prisma/schema.prisma [hunk 2]
```diff
@@ -48,9 +48,10 @@ model LocaleName {
 
 // Represents a language spoken by a healthcare provider
 model SpokenLanguage {
-  iso639_3 String @id
-  nameJa   String
-  nameEn   String
+  iso639_3   String @id
+  nameJa     String
+  nameEn     String
+  nameNative String
```
Comment (ann-kilzer) [inline prisma/schema.prisma:54]: for showing the language in its own language. E.g. German => Deutsch, Japanese => 日本語

File: src/mockData/mockData.ts [hunk 3]
```diff
@@ -48,7 +46,7 @@ const facilities: Array<Facility> = [
                     }
                 ],
                 degrees: [<Degree>'DOCTOR_OF_MEDICINE'],
-                spokenLanguages: [<Language>'JAPANESE', <Language>'ENGLISH'],
+                spokenLanguages: [],
```
Comment (ann-kilzer) [inline src/mockData/mockData.ts:49]: Since we're moving away from mockData to a real database, I didn't bother updating these

**Review by theyokohamalife — APPROVED** [review]
Still trying to get this to work locally, but approving since we used it together on your machine.

File: prisma/schema.prisma [hunk 4]
```diff
@@ -2,7 +2,8 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider = "prisma-client-js"
+  provider      = "prisma-client-js"
+  binaryTargets = ["native", "linux-arm64-openssl-3.0.x", "linux-musl"]
```
Comment (ann-kilzer) [inline prisma/schema.prisma:6]: @ermish take a look

**Pull request merged**